typings/index.d.ts

```diff
@@ -711,6 +711,7 @@ declare module 'discord.js' {
 		public shardID: number;
 		public splash: string | null;
 		public readonly systemChannel: TextChannel | null;
+		public readonly systemChannelFlags: SystemChannelFlags;
```

Comment:
```suggestion
		public systemChannelFlags: Readonly<SystemChannelFlags>;
```

The `readonly` modifier is used for getters, the `Readonly<T>` type should be used for Permissions where Permissions#freeze() is called